index.js

```diff
@@ -26,12 +26,14 @@ module.exports = function (opts) {
 		var full =
 			'\n' +
 			(file.cwd ? 'cwd:   ' + prop(tildify(file.cwd)) : '') +
+			'\nprocess.cwd: ' + prop(process.cwd()) +
+			(file.relative ? '\nrelative:   ' + prop(tildify(file.relative)) : '') +
 			(file.base ? '\nbase:  ' + prop(tildify(file.base)) : '') +
 			(file.path ? '\npath:  ' + prop(tildify(file.path)) : '') +
 			(file.stat && opts.verbose ? '\nstat:' + prop(stringifyObject(file.stat, {indent: '       '}).replace(/[{}]/g, '').trimRight()) : '') +
 			'\n';
 
-		var output = opts.minimal ? prop(path.relative(process.cwd(), file.path)) : full;
```

Comment:
The minimal view is meant to show you which files are running through the pipe. It could really be just full paths, but it's made relative to the cwd to be more readable. Use verbose view if you need more detailed info.